bzt/resources/selenium_extras.py

```diff
@@ -145,3 +145,69 @@ def get_locator(self, locators):
         # restore the implicit wait value
         self.driver.implicitly_wait(self.timeout)
         return locator
+
+
+class DialogsManager:
+    """
+    Provides additional methods for working with Dialogs that are not available in the Python Webdriver.
+    These JavaScript functions are taken from the Java Selenium Webdriver repository
+    """
+
+    def __init__(self, driver):
+        self.driver = driver
+
+    def replace_alerts(self):
```

Comment:
could you explain the purpose of methods (in comments), especially vague ones like this.

Reply:
Sure, I agree it may not be so clear... I mostly kept the same methods as in official Selenium repo.